index.html

```diff
@@ -8,51 +8,232 @@
     <link rel="stylesheet" href="styel.css" />
   </head>
   <body>
-    <header class="nav-container">
-      <div class="wrapper">
+    <div class="wrapper">
+      <header class="nav-container">
+        <div class="logo">
+          <a href="#">My Logo</a>
+        </div>
         <nav>
-          <a href="#" class="logo">Logo</a>
-          <img class="image" src="Enabled.svg" alt="" />
+          <figure>
+            <img class="image" src="Enabled.svg" alt="menu" />
+          </figure>
         </nav>
-      </div>
-    </header>
-    <div class="header-container">
-      <div class="wrapper">
-        <section>
-          <div class="typography">
-            <h1>
-              I’m Mahdia <br />
-              Glad to see you!
-            </h1>
-            <p>
-              I’m a software developer! I can help you build a product , feature
-              or website Look through some of my work and experience! If you
-              like what you see and have a project you need coded, don’t
-              hestiate to contact me.
+      </header>
+      <section class="intro">
+        <h1 class="title">I’m Mahdia, Glad to see you!</h1>
+        <p>
+          I’m a software developer! I can help you build a product , feature or
+          website Look through some of my work and experience! If you like what
+          you see and have a project you need coded, don’t hestiate to contact
+          me.
+        </p>
+
+        <div class="social-media">
+          <h5>LETS CONNECT</h5>
+          <ul>
+            <li>
+              <a href=""><img src="twitter.svg" alt="" /></a>
+            </li>
+            <li>
+              <a href=""><img src="linked.svg" alt="" /></a>
+            </li>
+            <li>
+              <a href=""><img src="meduimsvg.svg" alt="" /></a>
+            </li>
+            <li>
+              <a href=""><img src="github.svg" alt="" /></a>
+            </li>
+            <li>
+              <a href=""><img src="angellist.svg" alt="" /></a>
+            </li>
+          </ul>
+        </div>
+      </section>
+      <section class="works">
+        <div class="cards">
+          <img src="Snapshoot.svg" alt="" />
+          <div class="projects">
+            <div class="project-section">
+              <h1 class="project-title">Tonic</h1>
```

Comment:
- [ ] Kindly make sure you have only one h1 tag in your title. For the rest of the website use h2, h3, h4,h5 or h6. 
Replace all the h1 tags with other hx tags except the title.